x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.tsx

```diff
@@ -346,8 +384,74 @@ export const EndpointList = () => {
           );
         },
       },
+      {
+        field: '',
+        name: i18n.translate('xpack.securitySolution.endpoint.list.actions', {
+          defaultMessage: 'Actions',
+        }),
+        actions: [
+          {
+            // eslint-disable-next-line react/display-name
+            render: (item: HostInfo) => {
+              return (
+                <TableRowActions
+                  items={[
+                    <EuiContextMenuItem icon="logoSecurity" key="hostDetailsLink">
+                      <LinkToApp
+                        data-test-subj="hostLink"
+                        appId="securitySolution"
+                        appPath={`hosts/${item.metadata.host.hostname}`}
+                        href={`${services?.application?.getUrlForApp('securitySolution')}/hosts/${
+                          item.metadata.host.hostname
+                        }`}
+                      >
+                        <FormattedMessage
+                          id="xpack.securitySolution.endpoint.list.actions.hostDetails"
+                          defaultMessage="View Host Details"
+                        />
+                      </LinkToApp>
+                    </EuiContextMenuItem>,
+                    <EuiContextMenuItem icon="logoObservability" key="agentConfigLink">
+                      <LinkToApp
+                        data-test-subj="agentPolicyLink"
+                        appId="ingestManager"
+                        appPath={`#/policies/${
+                          agentPolicies[item.metadata.Endpoint.policy.applied.id]
+                        }`}
+                        href={`${services?.application?.getUrlForApp('ingestManager')}#/policies/${
```

Comment:
I just found that Ingest also has a set of methods that allows us to get the `path` to the different pages:

https://github.com/elastic/kibana/blob/3031e668060ad56124a0f191f1c03cea71532697/x-pack/plugins/ingest_manager/public/applications/ingest_manager/constants/page_paths.ts#L59-L84

This however, does not seem to be exported at the `ingest_manager/public` level, so we can't reference it just yet. but maybe we open a separate issue to use it and then refactor this here (and possibly other locations in our code) to use it?

cc:/ @jen-huang , @nchaulet 😬

Reply:
feel free to export and use this whenever :)